Sources/Workspace/DefaultPluginScriptRunner.swift

```diff
@@ -393,16 +393,36 @@ public struct DefaultPluginScriptRunner: PluginScriptRunner, Cancellable {
 
         // Optionally wrap the command in a sandbox, which places some limits on what it can do. In particular, it blocks network access and restricts the paths to which the plugin can make file system changes. It does allow writing to temporary directories.
         if self.enableSandbox {
-            command = Sandbox.apply(command: command, strictness: .writableTemporaryDirectory, writableDirectories: writableDirectories + [self.cacheDir], readOnlyDirectories: readOnlyDirectories)
+            var sandbox = SandboxProfile()
+            
+            // Allow writing in the temporary directory (whether or not it's overridden by TMPDIR).
+            sandbox.pathAccessRules.append(.writable(localFileSystem.tempDirectory))
```

Comment:
hard coding to localFileSystem okay in this case?

Reply:
Yes, because the only file system the plugin process has access to is the local file system.